code/ATMOSPHERICS/components/trinary_devices/filter.dm

```diff
@@ -1,6 +1,13 @@
+#define FILTER_NOTHING			-1
+#define FILTER_PLASMA			0
+#define FILTER_OXYGEN			1
+#define FILTER_NITROGEN			2
+#define FILTER_CARBONDIOXIDE	3
+#define FILTER_NITROUSOXIDE		4
+
 /obj/machinery/atmospherics/trinary/filter
 	icon_state = "filter_off"
-	density = 1
+	density = 0

```

Comment:
Oh goodness, forgot to mention that.
Yeah, since I was porting most of bay's atmospheric bits and bobs, I figured that was one of their features we sorely needed.